MODULE.bazel

```diff
@@ -0,0 +1,119 @@
+module(
+    name = "everest-framework",
+    version = "0.0.0",
```

Comment:
we should add this to the [check_version_consistency.py](https://github.com/EVerest/everest-framework/blob/main/.ci/build-kit/scripts/check_version_consistency.py) to keep it in sync with what we have in CMake.
I can do that quickly for this PR